dev/io.openliberty.xmlBinding.3.0.internal.tools/bnd.bnd

```diff
@@ -53,9 +54,10 @@ Import-Package: \
   !org.apache.xerces*, \
   !org.jvnet*, \
   !com.sun.mirror.*, \
-  !com.sun.codemodel.*,\ 
+  com.sun.codemodel.*,\
   !com.sun.source.tree.*, \
   !com.sun.source.util.*, \
+  !com.sun.xml.dtdparser.*, \
```

Comment:
In wanting to make sure xmlWS-3.0 and xmlBinding-3.0 started correctly post-changes to auto feature files, I found two unresolved import errors for these two packages. The DTD Parser might need to be brought in to fully support xmlBinding-3.0 at some point, but at this point I'm not sure if it's required.